src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -29,10 +29,10 @@
         </requiredProperty>
 
         <!-- DO NOT REFORMAT BECAUSE REPLACER PLUGIN WILL FAIL-->
-        <requiredProperty key="jslSpringbootVersion"><defaultValue>JSL-SPRINGBOOT-PLACEHOLDER</defaultValue></requiredProperty>
+        <requiredProperty key="jslSpringbootVersion"><defaultValue>1.0.4.20230221_041641_66ea6c71_develop</defaultValue></requiredProperty>
 
         <!-- DO NOT REFORMAT BECAUSE REPLACER PLUGIN WILL FAIL-->
-        <requiredProperty key="jslTatamiVersion"><defaultValue>JSL-TATAMI-PLACEHOLDER</defaultValue></requiredProperty>
+        <requiredProperty key="jslTatamiVersion"><defaultValue>1.1.4.20230221_041625_ba9cc1a5_develop</defaultValue></requiredProperty>
```

Comment:
This looks a bit suspicious part 1

Reply:
It is replkaced by compilayion. Do not commit these changes.

Reply:
[9b0620e](https://github.com/BlackBeltTechnology/judo-jsl-springboot-archetype/pull/31/commits/9b0620e949946d45b3567e895c56a555455b4774)